internal/cmd/config/cmd.go

```diff
@@ -12,11 +13,25 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+type CorsConfig struct {
+	AllowedOrigins         []string                                  `yaml:"allowed-origins"`
+	AllowOriginFunc        func(origin string) bool                  `yaml:"allow-origin-func"`
+	AllowOriginRequestFunc func(r *http.Request, origin string) bool `yaml:"allow-origin-request-func"`
+	AllowedMethods         []string                                  `yaml:"allowed-methods"`
+	AllowedHeaders         []string                                  `yaml:"allowed-headers"`
+	ExposedHeaders         []string                                  `yaml:"exposed-headers"`
+	MaxAge                 int                                       `yaml:"max-age"`
+	AllowCredentials       bool                                      `yaml:"allow-credentials"`
+	OptionsPassthrough     bool                                      `yaml:"options-passthrough"`
+	Debug                  bool                                      `yaml:"debug"`
```

Comment:
I'm not sure if all of those options are needed.
We should only expose ones we want

Reply:
Resolved in commit: https://github.com/aerogear/graphql-link/pull/59/commits/f8eb6d5d2a1a99cfbfc865c88d53d13504cc1f93